src/Agents.jl

```diff
@@ -58,7 +58,7 @@ export Models
 
 # Update message:
 display_update = true
-version_number = "4.2"
+version_number = "4.3"
```

Comment:
If this is changed, the update message will trigger. If we don't want to trigger an update message, then either set `display_update = false` or leave the version unchanged.